contracts/src/Rollup.sol

```diff
@@ -210,7 +226,28 @@ contract Rollup is RollupBase {
     }
 
     /// @inheritdoc IRollup
-    function stake() external payable override {
+    function updateValidatorWhitelist(address[] calldata validatorWhitelistUpdate, bool[] calldata vals)
+        external
+        override
+        onlyOwner
```

Comment:
should there be a way to for validators to revoke their own permissions? 
might be useful in the case of compromised keys

Reply:
good idea